### PR TITLE
Add link to twitter avatar

### PR DESCRIPTION
**Pull request by makoto**
This adds link to twitter profile only if twitter profile is set. Didn't add it to header as header profile is him/herself.

### CODE REVIEW

File: src/components/User/TwitterAvatar.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+import React from 'react'
+import { getSocial } from '../../utils/parties'
+
+const TwitterAvatar = ({ social, avatar }) => {
+  console.log('social', social)
```
Comment (hiddentao) [inline src/components/User/TwitterAvatar.js]: Please remove

File: src/components/SingleEvent/EventInfo.js [hunk 2]
```diff
@@ -120,12 +121,7 @@ class EventInfo extends Component {
         </ContractAddress>
         <EventImage src={party.image || 'https://placeimg.com/640/480/tech'} />
         <Organiser>
-          <Avatar
-            src={`https://avatars.io/twitter/${getSocial(
-              party.owner.social,
-              'twitter'
-            ) || 'randomtwitter'}`}
-          />{' '}
+          <TwitterAvatar social={party.owner.social} avatar={Avatar}/>
```
Comment (hiddentao) [inline src/components/SingleEvent/EventInfo.js]: We shouldn't be passing the styled `Avatar` through. `TwitterAvatar` should import it and use it internally.

File: src/components/SingleEvent/Participant.js [hunk 3]
```diff
@@ -28,9 +30,10 @@ const ParticipantWrapper = styled('div')`
   align-items: center;
 `
 
-const TwitterAvatar = styled('img')`
+const Avatar = styled(DefaultAvatar)`
```
Comment (hiddentao) [inline src/components/SingleEvent/Participant.js]: Better to use `css` function of `react-emotion` and then pass through as a `className` -> https://emotion.sh/docs/css

Comment (jefflau) [reply src/components/SingleEvent/Participant.js]: Actually passing through className is standard practice with emotion styled components. As long as you are not actually writing the classnames yourself. You're just exposing the className so emotion can give it dynamic classNames.

https://emotion.sh/docs/styled#styling-any-component

File: src/components/SingleEvent/EventInfo.js (hunk 2 above)
Comment (jefflau) [reply src/components/SingleEvent/EventInfo.js]: Agreed

**Comment by hiddentao** [comment]
We should fix up this PR and do the necessary code changes, then get it in.

**Comment by hiddentao** [comment]
@jefflau Please approve

**Pull request merged**